### ISSUE TITLE
Visualization of CBF+WM - contour and template WM

### ISSUE THREAD

**Issue by jan-petr** [ExploreASL/ExploreASL#182]
For the visualization of the CBF alignment with the WM mask in the standard space, add two extra options:

x.bVisualQCCBFvsWMTemplate - if on, then produces an additional file that use the template WM instead of the individual WM - to be used when the WM segmentation is questionable.
x.bVisualQCCBFvsWMContour - if on, then produces the the WM as a contour, not a full ROI - 1 pixel thickness (contour calculated per slice basis).

These options work on top of the standard output and produce files with different names - so that it could be recognized. Once this functionality is done - we can discuss all the names and the exact logistics.

**Comment by jan-petr** [comment]
See the commit - check if you like the functionality and parameter names and let me know if these should be changed.

**Comment by HenkMutsaerts** [comment]
Looks good code-wise!
### But... :)
Funny though that you now do exactly the same as with the `bNativeSpace` thingy, at least from my perspective :)
The current functions we use:
* `xASL_vis_CreateVisualFig`: take one or multiple images to create a figure from a NIfTI, with or without the overlay. Also works on its own (outside ExploreASL, i.e. it is a self-contained function)
* `xASL_wrp_VisualQC_ASL`: this is a wrapper that communicates between ExploreASL and the function `xASL_vis_CreateVisualFig`

So the most transparent way is to add the contour option to the function, and in the wrapper state in which cases we want this option (i.e. the booleans you created). Instead, you added all functionality to the wrapper ;)
So the solution is to move the contour creation etc to the function, and keep the booleans (i.e. the ExploreASL wrapping part) in the wrapper, don't you agree?

### By the way
Why didn't you use the SPM function that I mentioned in the issue, is this less good? Or did you recreate a new issue and didn't see this :)

### Otherwise
* xASL_delete already checks for if exist, including gz support, so this can be shortened (by removing the `if exist()` part
* pGMC is a bit unclear, would replace by pGM_contour
